tasks/util/env.py

```diff
@@ -11,16 +11,29 @@
 AZURE_STORAGE_SKU = "Standard_LRS"
 AKS_CLUSTER_NAME = "faasm-cluster"
 
-# 4 4-core machines: kernels, lammps, and migration experiment w/ 4 parallel functions
-# AZURE_VM_SIZE = "Standard_D4_v5"
-# AKS_CLUSTER_NODE_COUNT = 4
-
-# 2 8-core machines: migration experiment w/ 8 parallel functions
-# AZURE_VM_SIZE = "Standard_D8_v5"
-# AKS_CLUSTER_NODE_COUNT = 2
+KUBECTL_BIN = join(PROJ_ROOT, "bin", "kubectl")
 
-# 1 16-core machine: LULESH experiment
-AZURE_VM_SIZE = "Standard_D16_v5"
+AZURE_PUB_SSH_KEY = "~/.ssh/id_rsa.pub"
```

Comment:
I am confused, how will this work now? Will it use the machine's key's for accessing the VM?

Reply:
I assume it just adds your public key to the authorized keys for the machine. I think it was doing this anyway but wasn't sure, plus I sometimes create new key pairs for stuff like this so wanted to make it configurable.